ani-cli

```diff
@@ -209,8 +224,14 @@ download_dir="${ANI_CLI_DOWNLOAD_DIR:-.}"
 quality="${ANI_CLI_QUALITY:-best}"
 case "$(uname)" in
 	Darwin*) player_function="${ANI_CLI_PLAYER:-iina}" ;;
-	*) player_function="${ANI_CLI_PLAYER:-mpv}" ;;
+	*) if uname -a | grep -q 'steamdeck';then
+        player_function="${ANI_CLI_PLAYER:-flatpak_mpv}"
+    else
```

Comment:
This (and android) should be integrated to the case statement, by branching on `uname -a` and rethinking the other cases. It requires some research, so I'll try to help with this when I have the time

Reply:
Sorry, this one is a mistake